test/run_tests_tls_netaccess

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+
+function test_tls_server {
+  msg=$1
+  suite=$2
+  version=$3
+  EXIT_CODE=test_tls_server_tmp_code
+  OSSL_STDO=test_tls_server_tmp_openssl_stdout
+  OSSL_STDE=test_tls_server_tmp_openssl_stderr
+  SCAPY_STDE=test_tls_server_tmp_scapy_stderr
+
+  echo "1" > $EXIT_CODE
+  ($TRAVIS_SUDO python tls_test_server.py "$msg" 2>$SCAPY_STDE; echo $? > $EXIT_CODE) &
```

Comment:
I tried to but results were not conclusive. In the end I ran `coverage` directly from my shell on `tls_test_server.py`, and it confirmed that (apart from the `__init__`) the code inside the automaton was not covered.